Automattic/jetpack
# Instant Search: support multi site search results

#### Changes proposed in this Pull Request:
* Second part of 2592-gh-Automattic/p2 previous part added support for sidebar static filters https://github.com/Automattic/jetpack/pull/19885
* This PR adds support for multi site result format (which contains site icon, site name, author name and reworked user friendly date string)
![Markup on 2021-06-14 at 2:17:04 PM](https://user-images.githubusercontent.com/2129455/121884192-4cb9fb00-cd1b-11eb-97cc-ec87ebde3dfa.png)



#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
- Apply D61900-code
- Navigate to an a8c p2
- Switching between "P2 Name" and "All Automattic" in the sidebar filters should change how search results look
- For "All Automattic" it should contain blog and author info
- Results title "Found ... results" should also change based on the group used to search in
